src/Dapplo.Confluence/GroupExtensions.cs

```diff
@@ -49,5 +49,35 @@ public static async Task<IList<Group>> GetGroupsAsync(this IGroupDomain confluen
             var response = await groupUri.GetAsAsync<HttpResponse<Result<Group>, Error>>(cancellationToken).ConfigureAwait(false);
             return response.HandleErrors()?.Results;
         }
+
+        /// <summary>
+        ///     Get the members of a group
+        /// </summary>
+        /// <param name="confluenceClient">IGroupDomain to bind the extension method to</param>
+        /// <param name="groupName">string with the name of the group to retrieve the members for</param>
+        /// <param name="pagingInformation">PagingInformation</param>
+        /// <param name="cancellationToken">CancellationToken</param>
+        /// <returns>List with Users</returns>
+        public static async Task<IEnumerable<User>> GetGroupMembersAsync(this IGroupDomain confluenceClient, string groupName, PagingInformation pagingInformation = null, CancellationToken cancellationToken = default)
+        {
+            pagingInformation ??= new PagingInformation
+            {
+                Limit = 200,
+                Start = 0
+            };
+            var groupUri = confluenceClient.ConfluenceApiUri
```

Comment:
Some people like to add group within groups, I find this to be the case quite a bit at my company.
I'm not sure if this code would automatically drill-through down to the user level, or if recursion may be required to drill down even further.